packages/react/src/components/Indicator/Indicator.tsx

```diff
@@ -23,6 +23,9 @@ export interface IndicatorProps extends UIComponentProps {
   /** The indicator can point towards different directions. */
   direction?: 'start' | 'end' | 'top' | 'bottom'
 
+  /** The indicator can point towards different directions. */
+  code?: 'arrow' | 'close'
```

Comment:
What if we will rename `direction` to `variation` and include there `close`?

```diff
  /** The indicator can point towards different directions. */
- direction?: 'start' | 'end' | 'top' | 'bottom'
+  variation?: 'start' | 'end' | 'top' | 'bottom' | 'close'

-   /** The indicator can point towards different directions. */
-  code?: 'arrow' | 'close'
```

Reply:
That was my first thought but the direction is more for how the indicator/icon should be rotated. For example if you have the arrow icon, it can be pointed to different directions... In that sense what will the close mean?

Reply:
Plus in rtl the directions for start and end are swapped...

Reply:
In the same the code below doens't have any sense:
```jsx
<Indicator code='close' direction='start' />
```

Reply:
An actually, it's more `clear`/`cross` than `close` because it doesn't _closes_ anything in context of usage. May be it will be more obvious if we will prefix existing values with `arrow-` 🤔 

```jsx
<Indicator symbol='cross' />
<Indicator symbol='arrow-start' />
```

Reply:
Also, I expect that we will need more symbols like `minus` for example...

Reply:
I am just a bit afraid, that the `start | end | top | bottom` are one set of values, which are related to one another ('start' is replaced with 'end'), and this may be reused with other indicators as well (not sure how these character usages will grow in the future). Now we can ignore or augment the characters if different direction is used. At this moment, as the indicator is pretty simple, I am okay with either implementation, I am just thinking about future changes that may be introduced.

Reply:
@miroslavstastny @kuzhelov @Bugaa92 let's here another opinion on this one as well.

Reply:
I am sorry but I don't like the solution in general.
1. According to doc, `An indicator is suggesting additional content next to the element it is used in.` That is not true here.
2. **Theme** should be able to define its own close icon. How can that be achieved with this approach?

We should decide if this is a critical thing. And if it is not, I would rather discuss a more general approach on how theme can define content.

Reply:
Then back to the other proposal we had (this component was introduced only because we don't have the set of icon which will be available in all themes). Can we then define in the typings of the themes, that the arrows and close icons will always be available and just use icon names directly? If we are okay with that, then we won't even need this component at all. @kuzhelov @miroslavstastny @layershifter

Reply:
Or we can define icons/symbols directly in styles of this component.

Reply:
I agree with @miroslavstastny that this component is just covering some missing parts from our API. If we have the icons we don't need any of that.. I would rather have the icons being required as those are required by the API of the components we have...

Reply:
+1 for @miroslavstastny 's comment and looks like we're going towards the right solution

It'd be good to define some sort of artifacts per themes in <key,value> pairs that can be reused in components styles; same as fontAwesome does for icons; not sure if we need more than icons at this point. It'd also be good to let users edit these

Reply:
Created alternative PR: https://github.com/stardust-ui/react/pull/1120 please take a look